ouroboros-consensus-cardano/src/shelley/Ouroboros/Consensus/Shelley/Eras.hs

```diff
@@ -196,34 +192,51 @@ defaultApplyShelleyBasedTx globals ledgerEnv mempoolState _wti tx =
       mempoolState
       tx
 
+defaultGetProposedProtocolVersion ::
+  ( EraPParams era,
+    ProtVerAtMost era 8
+  ) => Core.PParamsUpdate era -> Maybe ProtVer
+defaultGetProposedProtocolVersion proposal =
+    strictMaybeToMaybe $ proposal ^. ppuProtocolVersionL
```

Comment:
I tried to emulate the formatting of the function above (`defaultApplyShelleyBasedTx`), though I don't exactly understand the rules, and `run-stylish.sh` seems to have to effect no matter the indentation I use.  Please let me know if it's wrong (it looks weird to me)

Reply:
Looks perfect, thanks!